test/src/middlewares/authorization-verifier.test.js

```diff
@@ -6,12 +6,12 @@ const createAuthorizationVerifier = require('../../../src/middlewares/authorizat
 const roles = require('../../../src/helpers/constants/roles');
 
 
-const ENCRYPTED_TOKEN = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoiYWRtaW4iLCJ1c2VyIjp7ImlkIjoxLCJmaXJzdE5hbWUiOiJhZG1pbiIsImxhc3ROYW1lIjoiYWRtaW4iLCJwYXNzd29yZCI6ImFkbWluIiwicGhvbmVOdW1iZXIiOiIwMDAtNTgxLTU0ODMiLCJlbWFpbCI6ImFkbWluQG1vb2RsZS5jb20ifSwiaWF0IjoxNTMxODMzOTM4LCJleHAiOjE1NjMzNjk5Mzh9.eHP-EJqF9C8nXGPT_LwdqtXAltHynFJSEKP21q6HRms';
+const ENCRYPTED_TOKEN = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoidGVhY2hlciIsInVzZXIiOjEsImlhdCI6MTUzMTkxMDIxOSwiZXhwIjo4NjQxNTMxOTEwMjE5fQ.v0YZpn0beF5x7fqCy_0j0A1011PBhoz6ofKZVo2GPvo';
```

Comment:
This will will not work forever, if you need a token in your test just sign a token before, or mock the token validator